roamance/src/components/Icons.tsx

```
@@ -1,65 +1,65 @@
-export function FacebookIcon(props: React.SVGProps<SVGSVGElement>) {
+export function Github(props: React.SVGProps<SVGSVGElement>) {
   return (
     <svg
-      className={props.className || "h-5 w-5"}
-      fill="currentColor"
+      className={props.className || 'h-5 w-5'}
       viewBox="0 0 24 24"
-      aria-hidden="true"
+      fill="currentColor"
       {...props}
     >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)